bench-jmh/README.md

```diff
@@ -6,8 +6,18 @@ own jmh module)
 
 You can run them like:
 
-   project bench-jmh
-   jmh:run -i 3 -wi 3 -f 1 .*ActorCreationBenchmark
+```shell
+sbt shell
+pekko > project bench-jmh
+sbt:pekko-bench-jmh> jmh:run -i 3 -wi 3 -f 1 .*ActorCreationBenchmark
```

Comment:
add a link to `sbt-jmh`?
and it's `bench-jmh` not `pekko-bench-jmh` now.

and change `jmh:run` to `Jmh/run`

Reply:
Done.  

`pekko-bench-jmh` is just the real-world execution result, I guess it was the name of the sub-module `bench-jmh`